pkg/oc/cli/debug/debug.go

```diff
@@ -191,6 +193,7 @@ func NewCmdDebug(fullName string, f kcmdutil.Factory, streams genericclioptions.
 	cmd.Flags().StringVar(&o.NodeName, "node-name", o.NodeName, "Set a specific node to run on - by default the pod will run on any valid node")
 	cmd.Flags().BoolVar(&o.AsRoot, "as-root", o.AsRoot, "If true, try to run the container as the root user")
 	cmd.Flags().Int64Var(&o.AsUser, "as-user", o.AsUser, "Try to run the container as a specific user UID (note: admins may limit your ability to use this flag)")
+	cmd.Flags().StringVar(&o.Image, "image", o.Image, "Override the image used by the targeted container.")
```

Comment:
Maybe worth pointing this applies only to node debugging, and adding validation which ensure this is not set in other cases or at least a warning so we don't get issues where someone will complain that they set the image and it's doing nothing.

Reply:
It doesn't only apply to node debugging.  It is used in transform podForDebug.

Primary use case is you want to debug the volume mounts of a given pod but with a different image.